website/docs/d/pi_instances.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+
+subcategory: "Power Systems"
+layout: "ibm"
+page_title: "IBM: pi_instances"
+description: |-
+  List all the power virtual server instances for the respective cloud instance in the Power Virtual Server cloud.
+---
+
+# ibm_pi_instances
+Retrieve information about all Power Systems Virtual Server instances for the given cloud instance. For more information, about Power Virtual Server instances, see [getting started with IBM Power Systems Virtual Servers](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started).
+
+## Example usage
+
+```terraform
+data "ibm_pi_instances" "ds_instance" {
+  pi_cloud_instance_id = "49fba6c9-23f8-40bc-9899-aca322ee7d5b"
+}
+```
+
+**Notes**
+* Please find [supported Regions](https://cloud.ibm.com/apidocs/power-cloud#endpoint) for endpoints.
+* If a Power cloud instance is provisioned at `lon04`, The provider level attributes should be as follows:
+  * `region` - `lon`
+  * `zone` - `lon04`
+  
+  Example usage:
+  
+  ```terraform
+    provider "ibm" {
+      region    =   "lon"
+      zone      =   "lon04"
+    }
+  ```
+  
+
+## Argument reference
+Review the argument references that you can specify for your data source. 
+
+- `pi_cloud_instance_id` - (Required, String) The GUID of the service instance associated with an account.
+
+## Attribute reference
+In addition to all argument reference list, you can access the following attribute references after your data source is created. 
+
+`pvm_instances` - List of power virtual server instances for respective cloud instance.
```

Comment:
This should be in a bulleted list. And all below attributes should be nested a level further.